python/py-sqlparse/Portfile

```diff
@@ -4,12 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-sqlparse
-version             0.3.1
+version             0.4.1
```

Comment:
this version does not support PY27 anymore; restrict it to the latest supported version (likely 0.3.1) for the py27 subport